apps/course-authoring/src/renderer/pages/home/data.tsx

```diff
@@ -21,6 +21,7 @@ export const cards: CardProps[] = [
     title: 'New Blank Document',
     content: 'Start from scratch with no template applied.',
     btnTitle: 'Create New',
+    btnRoute: '/editor/structure',
```

Comment:
can you change this so that imports the route from the page, also we should be defaulting to the build page.

Reply:
Done.